tests/v2/validation/provisioning/rke2/provisioning_node_driver_test.go

```diff
@@ -228,6 +229,96 @@ func (r *RKE2NodeDriverProvisioningTestSuite) ProvisioningRKE2ClusterDynamicInpu
 	}
 }
 
+func (r *RKE2NodeDriverProvisioningTestSuite) ProvisioningRKE2CNICluster(provider Provider) {
+	providerName := " Node Provider: " + provider.Name
+	r.cnis = []string{"canal", "cilium", "calico", "multus,canal", "multus,cilium", "multus,calico"}
```

Comment:
Tim I just realized you weren't doing this, and I see the confusion. We set the cnis in the config file this is just being redundant please remove this. That's why I was asking if we should just add the pod status stuff to the other tests. Make sense?

Reply:
Yea makes sense. I know in the future, we do want to test other things that may be specific with cni providers, should I keep ProvisioningRKE2CNICluster func or just move everything into ProvisioningRKE2Cluster?

Reply:
In that case let's keep the function, I'm not against. But definitely remove line 234. Thank you

Reply:
sgtm, updated the pr